src/modules/simulation/gz_bridge/GZMixingInterfaceServo.cpp

```diff
@@ -64,8 +64,8 @@ bool GZMixingInterfaceServo::updateOutputs(bool stop_motors, uint16_t outputs[MA
 	for (auto &servo_pub : _servos_pub) {
 		if (_mixing_output.isFunctionSet(i)) {
 			gz::msgs::Double servo_output;
-			///TODO: Normalize output data
-			double output = (outputs[i] - 500) / 500.0;
+			double output = math::radians((double)outputs[i] / 10. - 180.);;
```

Comment:
Are you assuming +1 is always 180 degrees? This is not true in general. We need a parameter defined which maps the normalized angles to servo angles. (part of the reason why the +1 radian was left like that)

Reply:
I assume that we only use angles/rates within +-180 deg or deg/s.

I made a commit to put the scaling and offset as parameters and updated the description in the generate_params.py to explain how parameters should be set. Can I put an explanation elsewhere or is that enough?

Reply:
@Perrrewi I am not talking about the limits, but rather that the angles need to be configurable. For example, tiltrotors normally operate the servo angles 0-90 deg, which you could do with +-45, but it would be great if we can actually do it properly and configure it.

Reply:
@Jaeyoung-Lim I made an update with angle min max as parameters.

Reply:
I tried adding the parameters on the control surface parameters CA_SV_CS in the control allocator, but then some targets got out of flash memory.

For now I cannot see a good solution before changing the types from uints to floats as proposed here: https://github.com/PX4/PX4-Autopilot/pull/22488

Reply:
@Jaeyoung-Lim As far as I understand the Gazebo model defines only the limits of the angles and not the scale so we should define the scale in PX4 parameters in the gz_bridge.

> tiltrotors normally operate the servo angles 0-90 deg

`SIM_GZ_SV_MIN{n}` = 1800
`SIM_GZ_SV_MAX{n}` = 2700
would scale it correctly for that now right?

> which you could do with +-45

The default would now be +-45° which is lower than the previous +-1rad which was in some cases cropped by the gazebo model's limits.